src/App.js

```diff
@@ -1,12 +1,27 @@
 // src/App.js
 import React from 'react';
 import Calculator from './components/Calculator';
+import MainComponent from './components/MainComponent'; // Import MainComponent
 import './App.css';
 
 function App() {
   return (
     <div className="App">
       <Calculator />
+      <MainComponent />
```

Comment:
- [ ] Please always read the instructions carefully and adhere to the requirements, as per the instructions, you should directly call your component right next to your `Calculator` component in the *Main* component, now here the main component is your **App.js**, it doesn't mean that you should have to create `MainComponent.js` file for importing your quotes.

### Requirement
![Screenshot from 2023-09-14 15-52-32](https://github.com/Hsuwaddyhtat22/react-magician/assets/66476329/b41f5d65-b66e-4cc8-9881-116bb2d99535)